SWLOR.Game.Server/Feature/AbilityDefinition/Force/BenevolenceAbilityDefinition.cs

```diff
@@ -43,7 +43,7 @@ private void Impact(uint activator, uint target, int baseAmount)
                 effect = TagEffect(effect, BeneRegen);
                 ApplyEffectToObject(DurationType.Temporary, effect, target, duration);
             }
-            var willHeal = baseAmount + targetBonus * 4 + Random.D10(targetBonus * 3);
+            var willHeal = baseAmount + (targetBonus * 4) + Random.D4(targetBonus);
```

Comment:
Make sure the Perk definition is updated to reflect this change (if specifics are displayed). Update the Bible as appropriate too.

Reply:
None of this needs to be changed in definitions or the bible.